DeusEx/Classes/DynamicTeleporter.uc

```diff
@@ -0,0 +1,32 @@
+class DynamicTeleporter extends Teleporter;
+
+var float proxCheckTime;
+var float Radius;
+
+simulated function Tick(float deltaTime)
+{
+    local DeusExPlayer Player;
+    Super.Tick(deltaTime);
+    proxCheckTime += deltaTime;
+    if (proxCheckTime > 0.1)//every 100ms
+    {
+        proxCheckTime = 0;
+        foreach RadiusActors(class'DeusExPlayer', Player, Radius)
+        {
+            Touch(Player);
+        }
+    }
+}
+
+function Trigger(Actor Other, Pawn Instigator)
+{
+	Touch(Instigator);
+}
+
+defaultproperties
+{
+    bGameRelevant=True
+    bCollideActors=False
+    bStatic=False
```

Comment:
normal teleporters are static by default, so they can't be spawned, also they don't run the Tick function because they aren't bGameRelevant, and the collision detection they use is baked into the map so that's why I need to use the Tick function